lambda/src/test/scala/pricemigrationengine/migrations/DigiSubs2023MigrationTest.scala

```diff
@@ -243,6 +243,245 @@ class DigiSubs2023MigrationTest extends munit.FunSuite {
     //   - emailMaxNotificationLeadTime
     //   - emailMinNotificationLeadTime
 
+    assertEquals(thereIsEnoughNotificationLeadTime(cohortSpec, today, cohortItem1), false) // +30 days
+    assertEquals(thereIsEnoughNotificationLeadTime(cohortSpec, today, cohortItem2), false) // +31 days
+    assertEquals(thereIsEnoughNotificationLeadTime(cohortSpec, today, cohortItem3), true) // +32 days
+    assertEquals(thereIsEnoughNotificationLeadTime(cohortSpec, today, cohortItem4), true) // +33 days
+    assertEquals(thereIsEnoughNotificationLeadTime(cohortSpec, today, cohortItem5), true) // +34 days
+  }
+  test("quarterly") {
+
+    assertEquals(newPriceLookup("GBP", Monthly).toOption.get, BigDecimal(14.99))
+    assertEquals(newPriceLookup("GBP", Quarterly).toOption.get, BigDecimal(44.94))
+    assertEquals(newPriceLookup("USD", Annual).toOption.get, BigDecimal(249))
+
+    val account = Fixtures.accountFromJson("DigiSubs2023/quarterly/account.json")
+    val catalogue = Fixtures.productCatalogueFromJson("DigiSubs2023/quarterly/catalogue.json")
+    val subscription = Fixtures.subscriptionFromJson("DigiSubs2023/quarterly/subscription.json")
+    val invoicePreview = Fixtures.invoiceListFromJson("DigiSubs2023/quarterly/invoice-preview.json")
+
+    val zuoraRatePlan1 = subscriptionRatePlan(subscription).toOption.get // If we fail here, that's part of the plan
```

Comment:
If you expect this to fail, it would be best to wrap it in an intercept: https://scalameta.org/munit/docs/assertions.html#intercept

Reply:
Actually it should never fail. If it does it means that the fixture is broken subscription. Comment is misleading. Let me remove it :)